r/R/parquet.R

```diff
@@ -428,6 +428,12 @@ ParquetFileWriter <- R6Class("ParquetFileWriter",
   inherit = ArrowObject,
   public = list(
     WriteTable = function(table, chunk_size) {
+      assert_is(table, "Table")
+      parquet___arrow___FileWriter__WriteTable(self, table, chunk_size)
+    },
+    WriteBatch = function(batch, chunk_size) {
+      assert_is(batch, "RecordBatch")
+      table <- Table$create(batch)
       parquet___arrow___FileWriter__WriteTable(self, table, chunk_size)
```

Comment:
These methods are super close together, so I doubt they would drift, but just in case, would it make sense to call `WriteTable(table, ...)` here in case we add other assertions / changes to `WriteTable` and forget to copy them down here?

Reply:
That makes plenty of sense, thanks for noticing. Fixed in 2a8fb0d9bb1113173d5eee7e7517b2b4234300c4.